### PR TITLE
feat(Icon): add `alert`, `info`, `share-alt` and `microsoft-stream` icons to Teams theme

### PR DESCRIPTION
**Pull request by marst89**
Add `alert`, `info`, `share-alt` and `microsoft-stream` icons to Teams theme.

### CODE REVIEW

**Comment by marst89** [comment]
icons added 
image

image

image

image
****

File: packages/react/src/themes/teams/components/Icon/svg/icons/alert.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" className={classes.svg} viewBox="8 8 16 16">
```
Comment (miroslavstastny) [inline packages/react/src/themes/teams/components/Icon/svg/icons/alert.tsx]: @codepretty This icon is always outline, never filled:
image
Is it ok?

Comment (marst89) [reply packages/react/src/themes/teams/components/Icon/svg/icons/alert.tsx]: For my case is fine. Also, for this particular one I just exported this from the processed ones. I did not create this from scratch.

Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/icons/alert.tsx]: Here is the filled path

`<path class="filled" d="M23.7,20L17.6,9.7c-0.1-0.2-0.3-0.4-0.5-0.5C16.9,9.1,16.7,9,16.4,9c-0.2,0-0.5,0.1-0.7,0.2c-0.2,0.1-0.4,0.3-0.5,0.5 L9.2,20C9.1,20.2,9,20.4,9,20.7c0,0.4,0.1,0.7,0.4,1c0.3,0.3,0.6,0.4,1,0.4h12.1c0.4,0,0.7-0.1,1-0.4c0.3-0.3,0.4-0.6,0.4-1 C23.9,20.4,23.8,20.2,23.7,20z M16,12.6c0-0.3,0.2-0.5,0.5-0.5s0.5,0.2,0.5,0.5v4.7c0,0.3-0.2,0.5-0.5,0.5S16,17.6,16,17.3V12.6z M16.5,20.8c-0.4,0-0.8-0.3-0.8-0.8c0-0.4,0.3-0.8,0.8-0.8s0.8,0.3,0.8,0.8C17.3,20.4,16.9,20.8,16.5,20.8z"></path>`

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { TeamsSvgIconSpec } from '../types'
 import accept from './accept'
 import add from './add'
 import addParticipant from './addParticipant'
+import alert from './alert'
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]: I would rename to `exclamation-triangle`

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 3]
```diff
@@ -244,6 +250,7 @@ export default {
   search,
   send,
   settings,
+  'share-recording': shareRecording,
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]: `share-recording` is too specific for this icon, i would suggest naming it more generically like `share` or `share-alt` as font awesome does

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-alert.tsx [hunk 4]
```diff
@@ -1,12 +1,29 @@
...
+        />
+        <path
+          className={cx(teamsIconClassNames.filled, classes.filledPart)}
+          d="M23.7,20L17.6,9.7c-0.1-0.2-0.3-0.4-0.5-0.5C16.9,9.1,16.7,9,16.4,9c-0.2,0-0.5,0.1-0.7,0.2c-0.2,0.1-0.4,0.3-0.5,0.5 L9.2,20C9.1,20.2,9,20.4,9,20.7c0,0.4,0.1,0.7,0.4,1c0.3,0.3,0.6,0.4,1,0.4h12.1c0.4,0,0.7-0.1,1-0.4c0.3-0.3,0.4-0.6,0.4-1 C23.9,20.4,23.8,20.2,23.7,20z M16,12.6c0-0.3,0.2-0.5,0.5-0.5s0.5,0.2,0.5,0.5v4.7c0,0.3-0.2,0.5-0.5,0.5S16,17.6,16,17.3V12.6z M16.5,20.8c-0.4,0-0.8-0.3-0.8-0.8c0-0.4,0.3-0.8,0.8-0.8s0.8,0.3,0.8,0.8C17.3,20.4,16.9,20.8,16.5,20.8z"
+        />
+      </g>
     </svg>
   ),
   styles: {},
+  exportedAs: 'exclamationTriangle',
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-alert.tsx]: Should be `name`:

```suggestion
  exportedAs: 'exclamation-triangle',
```

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-msft-stream.tsx [hunk 5]
```diff
@@ -10,4 +10,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'msft-stream',
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-msft-stream.tsx]:
```suggestion
  exportedAs: 'microsoft-stream',
```

Can we use full name?

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 6]
```diff
@@ -217,6 +222,7 @@ export default {
   mic,
   'mic-off': micOff,
   more,
+  'msft-stream': msftStream,
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]:
```suggestion
  'microsoft-stream': microsoftStream,
```

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 7]
```diff
@@ -86,6 +88,7 @@ import messageSeen from './messageSeen'
 import mic from './mic'
 import micOff from './micOff'
 import more from './more'
+import msftStream from './msftStream'
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts]:
```suggestion
import microsoftStream from './microsoftStream'
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -29,6 +29,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Features
 - Add 'poll' and 'to-do-list' icons to Teams theme @natashamayurshah ([#1498](https://github.com/stardust-ui/react/pull/1498))
+- Add 'alert', 'info',  'share-recording' and 'msft-stream' icons to Teams theme @marst89 ([#1544](https://github.com/stardust-ui/react/pull/1544))
```
Comment (layershifter) [inline CHANGELOG.md]:
```suggestion
- Add `alert`, `info`, `share-alt` and `microsoft-stream` icons to Teams theme @marst89 ([#1544](https://github.com/stardust-ui/react/pull/1544))
```

**Review by layershifter — APPROVED** [review]

**Pull request merged**